ext/liquid_c/block.c

```diff
@@ -262,6 +282,112 @@ static tag_markup_t internal_block_body_parse(block_body_t *body, parse_context_
     return unknown_tag;
 }
 
+VALUE parse_single_binary_comparison(VALUE markup) {
```

Comment:
Only supports parsing 1 binary comparison, need to add more parsing code to support full conditions.